app/helpers/admin/worldwide_organisations_helper.rb

```diff
@@ -68,4 +68,8 @@ def worldwide_office_shown_on_home_page_text(worldwide_organisation, worldwide_o
   def ordered_worldwide_offices(worldwide_organisation)
     ([worldwide_organisation.main_office] + [worldwide_organisation.home_page_offices] + [worldwide_organisation.other_offices]).flatten.uniq
   end
+
+  def any_translated_worldwide_offices?(worldwide_organisation)
+    worldwide_organisation.offices.map(&:contact).any? { |contact| contact.non_english_localised_models([:contact_numbers]) }
```

Comment:
I have checked this..but if we put outside the heading it will appear if no translated block is there like 
![image](https://github.com/alphagov/whitehall/assets/131259138/4db4ee59-d3e6-4363-92b4-d1b27667bba7)

May be we can discuss on Monday

Reply:
ah good catch it was returning nested arrays which are truthy `[[],[],[]]`

Reply:
ok great!